src/main/java/org/broadinstitute/hellbender/utils/ExtraMathUtils.java

```diff
@@ -0,0 +1,19 @@
+package org.broadinstitute.hellbender.utils;
+
+/**
+ * Extra MathUtils that should be moved to gatk-public
+ * Created by davidben on 1/22/16.
+ */
+public class ExtraMathUtils {
```

Comment:
Can we name this something like `GATKProtectedMathUtils`?  We're starting to have so many math utils classes it gets very confusing which ones are which.